docs/examples/example_01_hello_world.cpp

```diff
@@ -14,11 +14,28 @@
 
 #include <gtest/gtest.h>
 
+#include "cetl/pf17/sys/memory_resource.hpp"
+#include "cetl/pf17/byte.hpp"
+#include "cetl/variable_length_array.hpp"
+
+#include "example/foo_1_0.hpp"
```

Comment:
Did you also have an example of including a regulated data type?

Reply:
Wow. It's like you two don't trust me ;-) Yah. Will add...